4.x/typescript/Recenter/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Recenter",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
Can remove since there is no index.js. Can also remove empty description and keywords.

Reply:
Thanks Dasa, just one big copy/paste at times :-)